config/filter.d/mattermost.conf

```diff
@@ -0,0 +1,14 @@
+# Fail2Ban filter to match failed userlogin attempts as notified by mattermost's logfile
+#
+# Mattermost has to be configured to use timestamp at the start of the line. This is done in 'config/config.json'.
+# "LogSettings -> FileJson" needs to be set to 'false'
+# log lines should then look like this:
+# 2019-07-09T09:29:38.836+0200  error   mlog/log.go:172 Unable to find an existing account matching your credentials. This team may require an invite from the team owner to join.      {"path": "/api/v4/users/login", "request_id": "abc123", "ip_addr": "123.123.123.123", "user_id": "", "method": "POST", "err_where": "GetUserForLogin", "http_code": 400, "err_details": ""}
```

Comment:
The test file (belonging to the folder [fail2ban/tests/files/logs](https://github.com/fail2ban/fail2ban/tree/0.11/fail2ban/tests/files/logs)) with JSON describing each failure the filter matching (and possibly one not matching) is mandatory in the PR